### DISCUSSION TITLE
How do I get authToken?

### DISCUSSION THREAD

**Discussion by sureshvv** [dyte-io/html-samples#2] [Q&A] [answered]
I have registered. I have the Organization Id & API key.

**Comment by roerohan** [comment, marked as answer]
Hi @sureshvv 
The organization ID and API key can be obtained from our [Developer Portal](https://dev.dyte.io). You can sign up at https://dev.dyte.io and create an organization. You can head to the `API Keys` section on the sidebar to get the details you require, as you can see in the image below.

![image](https://github.com/dyte-io/html-samples/assets/42958812/18163230-e7bc-4a80-941f-76492736a74a)

Once you have the organization ID and API key, you can [create a meeting](https://docs.dyte.io/api#/operations/create_meeting), and then [add a participant](https://docs.dyte.io/api#/operations/add_participant) to the meeting. This will give you the auth token for the participant that you added.

We've been working on a Chat-GPT like AI bot that can help you with questions like these. You can check it out on our [Documentation Portal](https://docs.dyte.io) (check out the items on the right side of the top bar).

![image](https://github.com/dyte-io/html-samples/assets/42958812/955c923b-ae43-4acb-9488-663b289a67a5)

![image](https://github.com/dyte-io/html-samples/assets/42958812/f24035b7-d621-4d43-b486-ec3f5324023b)
